ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,199 @@
+
+![Process actions](/Assets/Screenshots/ActindoWorkFlow/Workflows/ProcessActions.png "[Process actions]")
+
+1. Click the *Actions* tab in the bottom part of the *Process ID* view.   
+  The *Actions* tab is displayed.
+
+  > [Info] By default, the *Actions* tab is preselected.
+
+  [comment]: <> (By default? Actions Reiter scheint standardmäßig ausgewählt zu sein, wenn man einen Prozess aus der Liste anklickt. Das heißt, erster Schritt etwas überflüssig)
+
+2. Click the *Status* drop-down list and select the **All** option.    
+  All actions, regardless of their status, are displayed.
+
+  > [Info] By default, the *Actions* tab lists all actions contained in the selected process.
+
+[comment]: <> (OLI: wenn eine Action/Token in einem Prozess ausgewählt werden, wird es als Filtereinstellung übernommen. Diese Filtereinstellung bleibt erhalten, auch man zu einem anderen Prozess wechlselt, daher sieht man nicht alle Actions, auch wenn auf "All" klickt. Bug oder erwartetes Verhalten?)
+
+3. Check the actions displayed in the diagram.  
+The workflow process diagram displays the actions in different colours depending on their current status:
+
+  - Active (blue)
+  - Done (green)
+  - Waiting (white)
+  - Error (red)
+  - Dead (black)
+
+ [comment]: <> (Andere möglichen Status? Vgl. mit Status in Actions Reiter: Active, Done, Error, Error; automatic retry, Suspended, Process aborted. Farben stimmen oder ändern sich noch?)
+
+ [comment]: <> (Oli: Statuses Error; automatic retry, Suspended and Process aborted. Wo stellt man "automatic retry" ein?)
+
+4. Click the action to be checked in the diagram.  
+The selected action is displayed in the *Actions* tab.
+
+5. Check the action status in the *Status* column and any further relevant information about the action displayed in the following columns:
+
+  - *Tries*: The number of times an action has been tried.
+  - *Maximal retries after error*: The number of times an action must be retried after error.
+  - *Queue type*: The *Default* queue type is normally selected. For detailed information about configuring queue types, see [Configure the queue types](#to-be-competed).
+
+    > [Info] A different queue type can be selected by editing an action, see [Edit an action](01_ManageWorkflows.md#edit-an-action).
+
+  - *Defer until*: The next point in time when the action is being tried.
+
+[comment]: <> (Oli: Is it possible to configure it somehow, i.e. Tries, Maximal retries after error and Defer until?)
+
+6. If an error is displayed, a troubleshooting procedure can be performed. For detailed information, see [Troubleshooting](ActindoWorkFlow/Troubleshooting/00_Troubleshooting.md).
+
+  > [Info] The *Token* tab displays detailed information about the tokens being processed in the actions. For detailed information about tokens, see [Check the token status and content](#check-the-token-status-and-content).
+
+7. Check the *Logs* tab for further details on the error displayed.
```

Comment:
hier nachhören, was in den Logs angezeigt wird -> ein error ist ja nicht in der Procedure vorgesehen, daher müsste das dann eher in ner Info eingebunden werden